packages/discord.js/typings/index.d.ts

```diff
@@ -5645,6 +5688,36 @@ export type GuildTemplateResolvable = string;
 
 export type GuildVoiceChannelResolvable = VoiceBasedChannel | Snowflake;
 
+export interface GuildOnboardingEditOptions {
+  prompts: GuildOnboardingPromptData[];
+  defaultChannels: ChannelResolvable[];
+  enabled: boolean;
+  mode: any;
+}
+
+export interface GuildOnboardingPromptData {
+  title: string;
+  singleSelect: boolean;
+  required: true;
+  inOnboarding: true;
```

Comment:
```suggestion
  required: boolean;
  inOnboarding: boolean;
```

as it doesnt have to?